### DISCUSSION TITLE
PRISM Blank App Template does not produce and app that compiles

### DISCUSSION THREAD

**Discussion by stevensrf11** [PrismLibrary/Prism#2384] [WPF]
Crap
When choosing the PRISM Blank App Template to create a .NetFramework App the project produce does not compile when I choose Unity.
Something about the Unity Package  version and I was told to some thing in the Package props

: The package reference 'Prism.Unity' should not specify a version.  Please specify the version in 'G:\prism folders\prism8\Prism-master\Packages.props' or set VersionOverride to override the centrally defined version.

**Comment by brianlagunas** [comment]
Works on my machine 😄

**Comment by stevensrf11** [comment]
Hmm lucky you.

I tried in the latest version of VS 2019.

Oh well no big loss

 
 

Sent: Monday, April 12, 2021 at 9:26 AM
